runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -346,6 +354,11 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotGauntletLoot() && chatMessage.equals(GAUNTLET_LOOTED_MESSAGE))
```

Comment:
Since the chest loot message is pretty ambiguous, this should also check that the player is next to the gauntlet chest by way of checking that the player is in the gauntlet chest region.

Reply:
Got ya. I pulled the location from the existing loot tracker regionId if that's alright

https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java#L134-L137